<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оконное приложение, управляющее БД</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableViewEmployees">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxAddRecord">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Добавление записи произвольного содержания в базу данных</string>
    </property>
    <widget class="QLabel" name="labelOccupation">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Род занятий</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFirstName">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxOccupation">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>сотрудник</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>студент</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEditFirstName">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddRecord">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить запись</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLastName">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxYearOfAdmission">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>2018</number>
     </property>
     <property name="value">
      <number>2010</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelYearOfAdmission">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Год поступления</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxDeleteRecord">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Удаление записи с заданным номером из базы данных</string>
    </property>
    <widget class="QLabel" name="labelID">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeleteRecord">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить запись</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxSort">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>440</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Сортировка таблицы</string>
    </property>
    <widget class="QComboBox" name="comboBoxSortColumn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Идентификатор</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Имя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Род занятий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Год поступления</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButtonSortData">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сортировка</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSortOrder">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>по возрастанию</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по убыванию</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxFiltration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Фильтрация</string>
    </property>
    <widget class="QComboBox" name="comboBoxFilterColumn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Идентификатор</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Фамилия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Имя</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Род занятий</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Год поступления</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEditFilterValue">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxFilterCondition">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>=</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>≠</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>≤</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>≥</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButtonFilter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Запрос</string>
     </property>
    </widget>
    <widget class="Line" name="lineInFiltration1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonYearOfAdmissionGreaterThan2010">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Год поступления &gt; 2010</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonYearOfAdmissionLessThan2011">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Год поступления &lt; 2011</string>
     </property>
    </widget>
    <widget class="Line" name="lineInFiltration2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowAllData">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывести всё</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreateNewDatabase"/>
    <addaction name="actionOpenDatabase"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionCreateNewDatabase">
   <property name="text">
    <string>Создать новую базу данных…</string>
   </property>
  </action>
  <action name="actionOpenDatabase">
   <property name="text">
    <string>Открыть созданную ранее базу данных…</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
